centreon/packaging/scripts/centreon-web-postinstall.sh

```diff
@@ -108,6 +108,16 @@ fixSymfonyCacheRights() {
   fi
 }
 
+fixCentreonCronPermissions() {
```

Comment:
you should use this function only for upgrades
for fresh install, set these rights directly in yam file

Reply:
updated